#!/usr/bin/env python3

PACKAGE = "pid_tunning"
from dynamic_reconfigure.parameter_generator_catkin import*
gen = ParameterGenerator()
gen.add("motor_speed",double_t,0,"motor control",0,-250,250)
gen.add("P",double_t,0,"motor control",0,-10,10)
gen.add("I",double_t,0,"motor control",0,-10,10)
gen.add("D",double_t,0,"motor control",0,-10,10)

exit(gen.generate(PACKAGE,"pid_tunning","motor_control"))